docs/content/Developer-Tools/Developer-Playground.md

```diff
@@ -11,14 +11,39 @@ applications from a variety of templates for different frontend frameworks and
 data visualization libraries.
 
 Developer Playground is available on `http://localhost:4000` when Cube.js is run
-in [development mode][link-dev-mode]
+in [development mode][link-dev-mode].
 
 Here's an example of building a query and plotting the results in Developer
 Playground:
 
-![](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Developer-Tools/playground.gif)
+<div
+  style="text-align: center"
+>
+  <img
+  alt="Playground Query and Results Example"
+  src="https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Developer-Tools/playground.gif"
+  style="border: none"
+  width="100%"
+  />
+</div>
 
-[link-dev-mode]: /configuration/overview
+## Using query tabs
+
+You can use query tabs to keep results of previous queries while still being
+able to make new queries. The query tabs at the top of the Developer Playground:
```

Comment:
Should this say "The query tabs are at the top of the..." or " See the query tabs at the top of the ..."?

Reply:
Hehe I forgot a few words 😀 Thanks for pointing it out 👍